src/components/CustomCard/custom-card.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import { Icon } from 'antd'
+
+class CustomCard extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {}
+    this.handleCoverClick = this.handleCoverClick.bind(this)
+  }
+
+  handleCoverClick(e) {
+    if(e.target.className.includes('website-link')) {
+      window.open(this.props.websiteLink, '_blank')
+    } else {
+      window.location = this.props.cardLink
+    }
+  }
+
+  render() {
+    let hasCover = this.props.cover.image.length > 0
+    let overlay = ''
+    if (hasCover) {
+      overlay = <div className='overlay'></div>
+    }
+  	return (
+  		<div className='custom-card'> 
+  			<div onClick={this.handleCoverClick} className='cover' style={this.props.cover.image.length > 0 ? {
+                  background: `url(${this.props.cover.image})`,
+                  backgroundSize: 'cover',
+                  backgroundColor: this.props.cover.bgColor,
+                } : {
```

Comment:
Please fix indentation. You can use eslint or [standard js](https://standardjs.com). We need to decide and set up a JS linter.